common/src/main/scala/explore/components/HelpIcon.scala

```diff
@@ -17,31 +17,29 @@ import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.common._
 
-final case class HelpIcon(id: Help.Id) extends ReactProps[HelpIcon](HelpIcon.component)
+final case class HelpIcon(id: Help.Id)
```

Comment:
@rpiaggio I converted this to a functional component for "fun", Does it seem ok?

Reply:
I wonder what your definition of fun is.

It looks great. I'm planning to generalize the implicit conversion into `react-common` by having `ReactProps` accept a functional component as parameter, but until we have that we have to manually provide the conversion.

Reply:
It's a `fun`ctional component.